components/org.wso2.carbon.identity.api.server.identity.governance/org.wso2.carbon.identity.api.server.identity.governance.v1/src/gen/java/org/wso2/carbon/identity/api/server/identity/governance/v1/model/MetaRes.java

```diff
@@ -0,0 +1,100 @@
+/*
+* Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Let's update the year correctly. You can refer to [1].
[1] - https://somindagamage.medium.com/adding-and-using-wso2-copyright-license-in-intellij-idea-8d172ce8c420

Reply:
Fixed in commit 37c3b01b50911acef42cdeb84eb9d2405b1c1c71

Reply:
make it 2021